CHANGELOG.md

```diff
@@ -48,5 +48,235 @@ Ref: https://keepachangelog.com/en/1.0.0/
+* (x/foundation) [\#62](https://github.com/Finschia/finschia/pull/62) add `x/foundation` module of lbm-sdk
+
+### Improvements
+
+### Bug Fixes
+* (app) [\#60](https://github.com/Finschia/finschia/pull/60) register authz module store key
+
+
+## [v0.3.0] - 2022-03-31
+
+### Bug Fixes
+* (build) [\#47](https://github.com/Finschia/finschia/pull/47) fix Docker build error
+
+### Features
+* (x/wasm) [\#41](https://github.com/Finschia/finschia/pull/41) upgrade x/wasm (merged original 0.19.0)
+* (x/upgrade) [\#42](https://github.com/Finschia/finschia/pull/42) add token module and bump cosmos-sdk v0.42.11
+
+
+## [v0.2.0] - 2022-02-04
+
+### Features
+* (x/upgrade) [\#33](https://github.com/Finschia/finschia/pull/33) To smoothen the update to the latest stable release, the SDK includes version map for managing migrations between SDK versions.
+* (x/consortium) [\#34](https://github.com/Finschia/finschia/pull/34) add feegrant, consortium and stakingplus module
+* (x/bank) [\#36](https://github.com/Finschia/finschia/pull/36) apply a feature that preventing sending coins to inactive contract (related to [finschia-sdk #400](https://github.com/Finschia/finschia-sdk/pull/400))
+
+### Improvements
+* (slashing) [\#31] (https://github.com/Finschia/finschia/pull/31) Apply VoterSetCounter
+
+## [v0.1.0] - 2021-11-01
+
+### Features
+* (app) Revise bech32 prefix cosmos to link and tlink
+
+### Improvements
+* (sdk) Use fastcache for inter block cache and iavl cache
+* (sdk) Enable signature verification cache
+* (ostracon) Apply asynchronous receiving reactor
+* (sdk) [\#21](https://github.com/Finschia/finschia/pull/21) Use lbm-sdk v0.43.0
+
+### Bug Fixes
+
+### Breaking Changes
+* (sdk) (auth) [\#16](https://github.com/Finschia/lfb/pull/16) Introduce sig block height for the new replay protection
+* (ostracon/sdk) [\#26](https://github.com/Finschia/lfb/pull/26) Use vrf-based consensus, address string treatment
+* (global) [\#10](https://github.com/Finschia/finschia/pull/10) Re-brand lfb to lbm
+
+### Build, CI
+* (build) [\#25](https://github.com/Finschia/finschia/pull/25) Fix localnet-start
+
 <!-- Release links -->
 [Unreleased]: https://github.com/Finschia/finschia/compare/v3.0.0...HEAD
+[v3.0.0]: https://github.com/Finschia/finschia/releases/tag/v3.0.0
+[v2.0.0]: https://github.com/Finschia/finschia/releases/tag/v2.0.0
+[v1.0.0]: https://github.com/Finschia/finschia/releases/tag/v1.0.0
+[v0.7.0]: https://github.com/Finschia/finschia/releases/tag/v0.7.0
+[v0.6.0]: https://github.com/Finschia/finschia/releases/tag/v0.6.0
+[v0.5.0]: https://github.com/Finschia/finschia/releases/tag/v0.5.0
+[v0.4.0]: https://github.com/Finschia/finschia/releases/tag/v0.4.0
+[v0.3.0]: https://github.com/Finschia/finschia/releases/tag/v0.3.0
+[v0.2.0]: https://github.com/Finschia/finschia/releases/tag/v0.2.0
+[v0.1.0]: https://github.com/Finschia/finschia/releases/tag/v0.1.0
+[gaia v4.0.4]: https://github.com/cosmos/gaia/releases/tag/v4.0.4
```

Comment:
This release link is not used in this file. So I think it can be removed. 
But please add original fork information below the v0.1.0 description

This release link is deprecated in this file. So I think it can be removed.
But please add at least some original fork information under the v0.1.0 description.
Like “This registory forked [gaia v4.0.4.](https://github.com/cosmos/gaia/releases/tag/v4.0.4)”

Reply:
Yeah you can check at `RELEASE_DESCR`